### ISSUE TITLE
Selfhosted Sentry: Github SSO internal error

### ISSUE THREAD

**Issue by infister** [getsentry/sentry#17522]
## Important Details

How are you running Sentry?

* [x] On-Premise docker [Version f7acdaa]
* [ ] Saas (sentry.io)
* [ ] Other [briefly describe your environment]

## Description

"Oops! Something went wrong" when trying to enable Github SSO in selfhosted sentry, after accepting App on Github. URL with error: 
`https://logger.********.net/auth/sso/?code=c42bfa7cc8070a3d****&state=84e5dcf7346e4cbb9913a62da4d1****`

Also, there is **missing** `client_id` in url heading to github sso (I had to paste it by myself).

tldr; Github integration works fine, SSO does not.

## Steps to Reproduce

1. Click Configure on Github SSO in organisation.
2. Accept app
3. You should see the error.

Good items to include here include:

```
JSONDecodeError: Expecting value: line 1 column 1 (char 0)
  File "django/core/handlers/exception.py", line 41, in inner
```

### What you expected to happen

Correct installation should happen.

### Possible Solution

No solution yet.

**Comment by pnghai** [comment]
@infister why did you close this? I'm getting the same error.
